Tests/PowerSkillTests/Helpers.cs

```diff
@@ -42,6 +42,7 @@ public static async Task<WebApiSkillResponse> QueryFunction(string inputText, Fu
                 Method = "POST"
             };
             var response = (OkObjectResult)(await function(request));
+            Console.WriteLine(JsonConvert.SerializeObject((WebApiSkillResponse)response.Value));
```

Comment:
Oops, looks like leftover debug code.

Reply:
By the way, when I do this sort of console debugging, I usually use `System.Diagnostics.Debug.WriteLine` instead of `Console`.